guides/coding_style/c/README.md

```diff
@@ -84,21 +97,29 @@ Functions should have the type on a seperate line proceeding the rest of the fun
 int
 main(int argc, char *argv[])
 {
+Block of Code
+" " "
+" " "
+}
 ```
 
-Return statements should have the value wrapped in parenthesis.
+Return statements should have the value wrapped in parenthesis.There are basically 4 types of jump statements:-
+*return
+*goto
+*continue
+*break
 
 ```C
 return (0);
 ```
 
 ## Comments
 
-Always use C style comments (`/* */`) and not C++ style comments (`//`). A sample is shown below.
+Always use C style comments (`/* */`) and C++ style comments (`//`). A sample is shown below.
```

Comment:
not was correct. In C, many compilers (older ones) do not support singe line comments.